src/styles/index.scss

```diff
@@ -1,3 +1,77 @@
+$hand-transform-origin: center bottom;
+
+@mixin center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin starting-position {
+  position: absolute;
+  bottom: 50%;
+}
+
 body {
+  @include center;
+
   margin: 0;
+  height: 100vh;
+}
+
+@keyframes infinite-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.stopwatch {
+  @include center;
+
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  position: relative;
+
+  &__minutes {
+    @include starting-position;
+
+    height: 20vmin;
+    width: 3vmin;
+    animation: infinite-spin 3600s steps(60) infinite;
+    transform-origin: $hand-transform-origin;
+    background-color: #0700ff;
+    z-index: 1;
+  }
+
+  &__seconds {
+    @include starting-position;
+
+    animation: infinite-spin 60s linear infinite;
+    transform-origin: $hand-transform-origin;
```

Comment:
It's better to use a variable for `background-color` as well. It will make your code more maintainable.